### ISSUE TITLE
Add example with a relic recipe

### ISSUE THREAD

**Issue by mbomb007** [MemeMayhem/ModExamples#32]
I looked at the examples for adding relics. I don't see any relic recipes. Is there one in a different mod that I'm missing, or could one be added?

I am getting ideas for relics that have a recipe.

**Comment by MajorMajorMajor6** [comment]
working on it, will add it to next version later today

**Comment by mbomb007** [comment]
Thanks. I did just notice that the relics page has a mention of the recipe, such as

```
id: happy_face | display_name: Happy Face | formula: fast_food + drooling_face
```

But it's not clear to me if the formula is a string or what.

**Comment by MajorMajorMajor6** [comment]
Ran into some unexpected issues with UI layer.. would need a bit more time to figure out what's going on.

**Comment by MajorMajorMajor6** [comment]
added a new example for relic combos, will be available in today's release, (0.20.136+)
https://github.com/MemeMayhem/ModExamples/blob/main/Basic-NewRelicCombo/additions/scripts/Trigger.lua.txt#L6
